12.8-2.3/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM postgres:12.8
+
+ENV PLV8_VERSION=2.3.15 \
+    PLV8_SHASUM="8a05f9d609bb79e47b91ebc03ea63b3f7826fa421a0ee8221ee21581d68cb5ba"
+
+RUN buildDependencies="build-essential \
```

Comment:
Some suggestions on this approach:

I like to stick `buildDependencies` into their own variable within the file, e.g. a `VAR` directive here.

Having each command be declared as it's own `RUN` stanza allows Docker to "cache" them. This way, when you're rebuilding or testing stuff, you don't need to run every single thing in this block, getting you a faster build.

Longer-term, this kind of stuff can be separated into layered images, making the builds even faster.